components/floatingactionbutton/positions.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
Consider `position-and-alignment.md` for better SEO or or just `position.md` for future-proofness.